normandy/selfrepair/static/js/normandy_driver.js

```diff
@@ -79,7 +129,22 @@ let Normandy = {
                 options.learnMoreMessage,
                 options.learnMoreUrl
             );
+
+            resolve(emitter);
```

Comment:
Returning an event emitter doesn't really help; we still have to call `showHeartbeat` before returning, so it might emit an event early before we return.

If this issue occurs, all that happens is that the timestamp for when the dialog was shown to the user is incorrect. Given that this is unlikely (In all my testing I've never once run into the issue of showHeartbeat emitting an event before the emitter listeners get registered), I think it's acceptable to just not handle this case. It'll be pretty clear from our data whenever this actually becomes an issue.